### DISCUSSION TITLE
Graphic comparison of the day before

### DISCUSSION THREAD

**Discussion by estrema-marco** [plausible/analytics#2559] [Anything Else]
Hi, it would be interesting in the daily view to have the graphic comparison of the day before or of 7 days before.
A very useful display is shown by clicky, where the two graphs are overlaid (the one from the day before in a lighter color)
![image](https://user-images.githubusercontent.com/119961694/210405299-9ca9c5e3-c0c5-428d-b978-508cb5b5337b.png)

and from the drop-down menu you can choose more comparisons
![image](https://user-images.githubusercontent.com/119961694/210405749-34e0c245-a2d5-42b5-9f25-d50a48dc3ada.png)

Also, the percentage data on the day before would make more sense if calculated relative to the same time the day before.
![image](https://user-images.githubusercontent.com/119961694/210406905-ccc4bf49-a646-41ee-9c1f-e6c207ead1cf.png)

What do you think?

**Comment by metmarkosaric** [comment]
**Update:** You can now compare your stats over time! Select "**Compare**" in the date picker or press the "**X**" key on your keyboard to start comparing. Learn more about the comparisons here: https://plausible.io/docs/compare-stats.
